plugins/woocommerce/client/blocks/tests/e2e/utils/test.ts

```diff
@@ -168,6 +169,22 @@ const test = base.extend<
 		},
 		{ scope: 'worker', auto: true },
 	],
+	wpCoreVersion: [
+		async ( {}, use ) => {
+			const output = await wpCLI( 'core version' );
+			const version = output.stdout.trim().split( '\n' ).at( -1 ) ?? '';
+			const parsedVersion = Number.parseFloat( version );
```

Comment:
WordPress versions should always have only two numbers, correct? I'm asking because this code doesn’t catch ‘6.10’.

Reply:
Great point, this will start failing when we reach x.10 and above. I'll make sure it works as expected.

Reply:
Added an explanation in https://github.com/woocommerce/woocommerce/pull/56584/commits/2fc0965248bb1056d15e8bf171932d271adfe3c9